docker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:22.04
+
+RUN apt-get update \
+  && apt-get install -y python3 python3-pip python3-venv
```

Comment:
Please find the correct way to do this, which is to clean the cache and sources lists...